cli/remove.go

```diff
@@ -64,8 +64,8 @@ func RemoveRun(_ *cmd.RootCMD, c *cmd.CMD) {
 		}
 	}
 	file = utils.BasicFileOpen(utils.AddedFilesPath, os.O_WRONLY | os.O_TRUNC, 0644)
-	file.Close()
 	err := utils.DumpMap(contents, file)
+	file.Close()
```

Comment:
Any reason we're not closing this before DumpMap?

Reply:
It absolutely should, and I actually fixed that in the `fix bug in Remove()` commit, so I'm not sure why you it's still there